port-forward-fun.conf

```diff
@@ -1,19 +0,0 @@
-##################### Notes ########################
-#                                                  #
-#  So, you wanna run a single Node server, huh?    #
-#  Here's what the Apache server might look like   #
-#                                                  #
-#  This will assume an Express server is already   # 
-#                running on Port 3000              #
-#                                                  #
-####################################################
-
-<VirtualHost *:80>
-    ServerAdmin          admin@mycoolsite.com
-    ServerName           mycoolsite.com
-    ProxyPreserveHost    On
-    ProxyPass            "/" "http://127.0.0.1:3000/"
-    ProxyPassReverse     "/" "http://127.0.0.1:3000/"
-    ErrorLog             ${APACHE_LOG_DIR}/mycoolsite.log
-    CustomLog            ${APACHE_LOG_DIR}/mycoolsiteaccess.log combined
```

Comment:
This log stores access data such as requests, the IP addresses they came from, the exact date and time, the status code (eg; 200 SUCCESS or 404 NOT FOUND) and the browsers user agent.

You can also sometimes see search spiders in there too. Heck, while writing this, my page was crawled by [Yahoo!](https://help.yahoo.com/kb/search/SLN22600.html?impressions=true)

The standard I follow for this is sitenameaccess.log. You also should include the combined part too because ¯\_(ツ)_/¯